### PR TITLE
Updated curators worklist, added approvals, owner changes and cleaning up a bit.

### PR DESCRIPTION
**Pull request by garshol**
First overview of request feature

### CODE REVIEW

**Comment by garshol** [comment]
Er ikkje ferdig med Worklist, men har fått på plass det meste rundt godkjenning og bytte av eigar (trur eg). Hart lagt til ein del info i readme files. Da tar eg ferie!

File: 1223-curator_worklist/my_worklist.feature [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		- A Request's Host is it's Senders'
+
+	Future stuff:
+		- The Custodian needs to be notified when a Request is updated (a new Message or someone adopts it)
+
+	Background: 
+		Given Curator is loged in
+		And Curator opens My Worklist from main menu
+		And My Worklist contains Requests of all four types
+		And all Requsts are sendt by a User Affilliated at Curators Scope
```
[the remark this answers is not in the document]
Comment (garshol) [reply 1223-curator_worklist/my_worklist.feature]: Done

File: 1223-curator_worklist/my_worklist.feature [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  @357
+    @updated
+    #Replaces @1353, 1354, 2756 and 531
+  Scenario Outline: Curator views a Request type
+    When they click on Requests of type "<Type>"
+    Then they see a list of Requests displayed with:
+      | Request status           |
+      | Registration title       |
+      | Submitter name           |
+      | Date                     |
```
Comment (terjehell) [inline 1223-curator_worklist/my_worklist.feature:46]: Which date?

File: 1223-curator_worklist/my_worklist.feature [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+      | Begining of last message |
```
Comment (terjehell) [inline 1223-curator_worklist/my_worklist.feature:47]: Beginning

File: 1223-curator_worklist/my_worklist.feature [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+      | Owner                    |
```
Comment (terjehell) [inline 1223-curator_worklist/my_worklist.feature:48]: Owner name?

File: 1223-curator_worklist/my_worklist.feature [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+      | Support   |
+      | DOI       |
+      | Ownership |
+
+  @1252
+  @updated
+  Scenario: Curator views details of a Request
+    When they selects one Request
+    Then the Requests message is expanded
+    And possible message previews are displayed
```
Comment (terjehell) [inline 1223-curator_worklist/my_worklist.feature:63]: Hvilke felter skal vises fram?

**Review by terjehell — CHANGES_REQUESTED** [review]

File: 1230-doi_minting/doi.feature [hunk 6]
```diff
@@ -0,0 +1,125 @@
+Feature: DOI related scenarios moved from MVP feature
```
[the remark this answers is not in the document]
Comment (garshol) [reply 1230-doi_minting/doi.feature:1]: Dette er gamle scenario som er flytta frå mvp.feature "as is".
Eg skal finne tid til å sjå over dei, no flytter vi dei berre til rett plass.

**Comment by garshol** [comment]
Have updated as needed to accommodate comments from reviewers

File: 1223-curator_worklist/my_worklist.feature [hunk 7]
```diff
@@ -115,71 +141,36 @@ Feature: Curator opens My Worklist
...
+      | Changes Owner            |
+      | Declines change of owner |
+    Then the Curator is Assigned the Request
 
   @needJiraTag
-  Scenario: Curator changes scope of Work List
-    When the Curator opens the Work List
-    Then the Curator may change scope to any other subunit at the institution
-    And all Requests from Senders in this new subunit is listed
+  Scenario: Decide if an Approval, Support or DOI Requests is in a Curators Scope
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:163]: "Curator receives Requests in their scope and requests they have been assigned from outside their scope", which to me means the latter scenario is to be split out

File: 1223-curator_worklist/my_worklist.feature [hunk 8]
```diff
@@ -115,71 +141,36 @@ Feature: Curator opens My Worklist
...
-    Then the Users Request is displayed to the Curator
-
-
-
-
-
-
+  Scenario: Decide if an Ownership Requests is in a Curators Scope
+    Given the Request is of type "Ownership"
+    When the Affilliation of the Owner of the Resource associated with the Request is withing Curators Scope
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:175]: within

File: 1223-curator_worklist/my_worklist.feature [hunk 9]
```diff
@@ -115,71 +141,36 @@ Feature: Curator opens My Worklist
...
-    When the Requests' Resources' Host equals the Curators Host
-    And Curators scope on the Institution include the Resources' Owners' User Profile affilliation to the Institution
-    Then the Users Request is displayed to the Curator
-
-
-
-
-
-
+  Scenario: Decide if an Ownership Requests is in a Curators Scope
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:173]: Curator receives Ownership requests within their scope (but not ones they have previously responded to?)

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 10]
```diff
@@ -1,28 +1,55 @@
 Feature: Curator handles an Approval Request
 
+    In order to manage the amount of resources created
+    As a Editor (at a large Institution like NTNU)
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:4]: an Editor

Comment (brinxmat) [reply 1223-curator_worklist/my_worklist_approval.feature:4]: In all places

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 11]
```diff
@@ -1,28 +1,55 @@
...
+    I want to allow the User to publish resources
+
+	In order to reduce cost of training and secure high quality on all resources
+    As a Editor (at a small Institution Like NINA)
+    I want an Curator to approv all resources prior to publishing
+
+    In order to remove unnessesery work and frustration frome duplicate resources
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:11]: frome -> from

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 12]
```diff
@@ -1,28 +1,55 @@
...
+    As a Editor (at a large Institution like NTNU)
+    I want to allow the User to publish resources
+
+	In order to reduce cost of training and secure high quality on all resources
+    As a Editor (at a small Institution Like NINA)
+    I want an Curator to approv all resources prior to publishing
+
+    In order to remove unnessesery work and frustration frome duplicate resources 
+    As a Editor (at a medium Institution like SINTEF)
+    I want to allow Users to publish metadata resources, then a Curator approv the resource before the content is made public accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:13]: publicly

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 13]
```diff
@@ -1,28 +1,55 @@
...
+
+    Rules:
+     - The User got publisher privelige
+     - User got publisher privelige, but all content need a Curator Approval to be public accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:17]: privilege

Comment (brinxmat) [reply 1223-curator_worklist/my_worklist_approval.feature:17]: twice

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 14]
```diff
@@ -1,28 +1,55 @@
...
+		# Don't remove the potentially original embargo on the files
 
 	@needJiraTag
-	Scenario: Curator approve a Approval Request
-		Given Owner wish to publish Registration, all restrictions
-		And Editor has restricted publishing
-		When Curator approves the Request
-		Then the Request status is "Done"
-		And the Resource's status is "Published"
+	Scenario: Curator decline access to content
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:43]: declines

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 15]
```diff
@@ -1,28 +1,55 @@
...
-	Scenario: Curator approve a File Approval Request
-		Given Owner wish to publish Registration, file restrictions
-		And Editor has restricted publishing of files
-		When Curator approves the Request
-		Then the Request status is "Done"
-		And remove the extra embargo on files
-		# Resource is published and Landing Page is public
-		# Don't remove the potentially original embargo on the files
\ No newline at end of file
+	Scenario: Curator decline an Approval Request
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:52]: declines

File: 1223-curator_worklist/readme [hunk 16]
```diff
@@ -1,41 +1,24 @@
-The Worklist is the Curators main tool to keep an overview of Requests from the Users. Each request is a part of a Resource (drafted or published). All Curators have access to the Worklist and share responsibillity to answer Requests from Users. 
+The Worklist is the Curator's main tool to follow up on Requests from the Users. The Curators share the Worklist and the responsibility to follow up all Requests. Each Request is part of a drafted or published Resource.
 
-Curators Scope
-The Editor may have (down-) Scope a Curators responsibiblity to any formal part of the Institusion and thus reduce the group of Users this Curator is responsible to handle. The Curator may override this Scope to view and assist the other Curators (at the same Institution).
+Curator Scope
+The Editor can narrower a curator's responsibility to any formal part of the Institusion like an faculty, institute or a reserch group. The Curator may override this and view other Requests at the Institution and thus assist other Curators.
```
Comment (brinxmat) [inline 1223-curator_worklist/readme:4]: can narrow

File: 1223-curator_worklist/readme [hunk 17]
```diff
@@ -1,41 +1,24 @@
...
 
 Search the Worklist
-The Worklist offer a Seach that enable the Curator to locate Request, regardless of Type, but Results are viewed by Types. The search results may be filterd on the requests status.
+The search enables the Curator to locate Requests. Search results are displayd by Type and may be filterd by Status.
```
Comment (brinxmat) [inline 1223-curator_worklist/readme:7]: filtered

File: 1228-landing_page/anonymous_user_views_landing_page_for_registration.feature [hunk 18]
```diff
@@ -17,4 +17,4 @@ Feature: Anonymous User views Landing Page for Registration
             | Files                           |
             | DOI link                        |
             | Related Registrations           |
-            | License                         |
\ No newline at end of file
+            | License                         |n
```
Comment (brinxmat) [inline 1228-landing_page/anonymous_user_views_landing_page_for_registration.feature:20]: trailing "n"?

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: 1223-curator_worklist/my_worklist.feature (hunk 7 above)
Comment (garshol) [reply 1223-curator_worklist/my_worklist.feature:163]: Have duplicated the scenario and adjusted them.

File: 1223-curator_worklist/my_worklist.feature (hunk 9 above)
Comment (garshol) [reply 1223-curator_worklist/my_worklist.feature:173]: Duplicated the scenario and adjusted

**Review by brinxmat — DISMISSED** [review]

**Review by terjehell — APPROVED** [review]

File: 1223-curator_worklist/my_worklist.feature [hunk 19]
```diff
@@ -0,0 +1,185 @@
...
+      | Support   |
+      | DOI       |
+      | Ownership |
+
+  @1252
+  @updated
+  Scenario: Curator views details of a Request
+    When they selects one Request
+    Then the Requests is expanded
+    And previews messages are displayed chronology with:
```
[the remark this answers is not in the document]
Comment (brinxmat) [reply 1223-curator_worklist/my_worklist.feature]: chronologically

**Review by terjehell — DISMISSED** [review]

File: 1223-curator_worklist/my_worklist.feature [hunk 20]
```diff
@@ -0,0 +1,190 @@
...
+    Examples:
+      | Type      |
+      | Approval  |
+			# Må handtere at Editor kan tillate alle å publisere - kva skjer da med Approvals?
+      | Support   |
+      | DOI       |
+      | Ownership |
+
+  @needJiraTag
+  Scenario: Curator open a unassigned Request
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:87]: opens

File: 1223-curator_worklist/my_worklist.feature [hunk 21]
```diff
@@ -0,0 +1,190 @@
+Feature: Curator opens My Worklist
+
+  "@needJiraTag" is used in several files to enable me to locate all new scenarios that is'nt in jira.
+  "@updated" is used where I have changed a scenario, that exists in jira - but not updated it there yet.
+	#If a scenario is deleted, but still exists in Jira - it's noted as a comment
+
+  Vocabulary clearification:
+  - A named role is preferd over "they"
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:8]: preferred

File: 1223-curator_worklist/my_worklist.feature [hunk 22]
```diff
@@ -0,0 +1,190 @@
...
+  - Resource is prefered over Registration
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:9]: preferred

File: 1223-curator_worklist/my_worklist.feature [hunk 23]
```diff
@@ -0,0 +1,190 @@
...
+
+  Rules:
+  - A Editor grant a User the Curator role at an Institution, to serve all users at the Institution
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:12]: An Editor

File: 1223-curator_worklist/my_worklist.feature [hunk 24]
```diff
@@ -0,0 +1,190 @@
...
+  @needJiraTag
+  Scenario: Curator change Scope of there Worklist
+    When the Curator change Scope to the whole Institution or one or more subunits
+    Then the listed Requests is updated to match the new Scope
+
+  @357
+  @updated
+  #Replaces @1353, 1354, 2756 and 531
+  Scenario Outline: Curator views all Requests of a type
+    When Curator click on Requests of type "<Type>"
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:69]: clicks

File: 1223-curator_worklist/my_worklist.feature [hunk 25]
```diff
@@ -0,0 +1,190 @@
...
+    Then Curator see a list of Requests displayed with:
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:70]: sees

File: 1223-curator_worklist/my_worklist.feature [hunk 26]
```diff
@@ -0,0 +1,190 @@
...
+      | Approval  | Publish      |
+      | Approval  | Decline      |
+      | DOI       | Mint DOI     |
+      | DOI       | Decline      |
+      | Ownership | Change owner |
+      | Ownership | Decline      |
+
+ @needJiraTag
+  Scenario: User gets an answer to a Support Request
+    When the Curator sends an answer of type "Support"
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:128]: When the Curator answers a Support Request?

File: 1223-curator_worklist/my_worklist.feature [hunk 27]
```diff
@@ -0,0 +1,190 @@
...
+    Then the Request status is set to "Answered"
+    And the User can read the answer in My Messages
+
+  @needJiraTag
+  Scenario: User gets an answer to a Request
+    When the Curator writes an answer
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:134]: Convert to scenario outline

When the Curator answers a Request of <Type>

File: 1223-curator_worklist/my_worklist.feature [hunk 28]
```diff
@@ -0,0 +1,190 @@
...
+    And sends it to the User
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:135]: Following two lines then become redundant

File: 1223-curator_worklist/my_worklist.feature [hunk 29]
```diff
@@ -0,0 +1,190 @@
...
+    And the Request Type is:
+      | Approval  |
+      | DOI       |
+      | Ownership |
+    Then the User can see the answer in My Messages
+
+  @needJiraTag
+  Scenario Outline: Curator change Status on a Request
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:143]: changes a Request Status

File: 1223-curator_worklist/my_worklist.feature [hunk 30]
```diff
@@ -0,0 +1,190 @@
...
+    When Curator selects a new status "<Status>" on a Request
+    Then the status of the Request is set to "<Status>"
+    Examples:
+      | Status   |
+      | Archived |
+      | Deleted  |
+
+  @needJiraTag
+  Scenario: Curator receives assignment of responses to requests they have previously responded to
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:152]: Curator scope contains responses to requests to which they have previously responded (maybe?)

File: 1223-curator_worklist/my_worklist.feature [hunk 31]
```diff
@@ -0,0 +1,190 @@
...
+      | Declines change of owner |
+    Then the Curator is Assigned the Request
+
+  @needJiraTag
+  Scenario: Curator receives Requests in their scope
+    Given the Request is of type:
+      | Approval |
+      | Support  |
+      | DOI      |
+    When the Requests' Submitter is Affilliated within the Curators Scope
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:168]: Request's

File: 1223-curator_worklist/my_worklist.feature [hunk 32]
```diff
@@ -0,0 +1,190 @@
...
+      | Support  |
+      | DOI      |
+    When the Curator is assigned the Request
+    Then the Request is part of the Curators Worklist
+
+  @needJiraTag
+  Scenario: Curator receives Ownership requests within their scope
+    Given the Request is of type "Ownership"
+    When the Affilliation of the Owner of the Resource associated with the Request is within Curators Scope 
+    Then the Request is part of the Curators Worklist
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist.feature:184]: Curator's

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 33]
```diff
@@ -0,0 +1,55 @@
+Feature: Curator handles an Approval Request
+
+    In order to manage the amount of resources created
+    As an Editor (at a large Institution like NTNU)
+    I want to allow the User to publish resources
+
+	In order to reduce cost of training and secure high quality on all resources
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:7]: -space

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 34]
```diff
@@ -0,0 +1,55 @@
...
+    As an Editor (at a small Institution Like NINA)
+    I want an Curator to approv all resources prior to publishing
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:9]: a Curator to approve all resources prior to publishing

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 35]
```diff
@@ -0,0 +1,55 @@
...
+
+    In order to remove unnessesery work and frustration from duplicate resources
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:11]: unnecessary

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 36]
```diff
@@ -0,0 +1,55 @@
...
+    As an Editor (at a large Institution like NTNU)
+    I want to allow the User to publish resources
+
+	In order to reduce cost of training and secure high quality on all resources
+    As an Editor (at a small Institution Like NINA)
+    I want an Curator to approv all resources prior to publishing
+
+    In order to remove unnessesery work and frustration from duplicate resources 
+    As an Editor (at a medium Institution like SINTEF)
+    I want to allow Users to publish metadata resources, then a Curator approv the resource before the content is made publicly accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:13]: approve

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 37]
```diff
@@ -0,0 +1,55 @@
...
+
+    Rules:
+     - The User got publisher privilege
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:16]: has publisher privilege

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 38]
```diff
@@ -0,0 +1,55 @@
...
+     - User got publisher privilege, but all content need a Curator Approval to be publicly accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:17]: User has publisher privilege, but all content needs Curator Approval to be publicly accessible

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 39]
```diff
@@ -0,0 +1,55 @@
...
+     - Only the Curator got publisher privilege
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:18]: has, not got

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 40]
```diff
@@ -0,0 +1,55 @@
...
+
+	Background: 
+		Given a User that needs an approval to publish
+		And the User has created an Approval Request
+		And the Curator views the landing page of the Resource assosiated with the Request
+
+	@needJiraTag
+	Scenario: Curator approves an Approval Request to publish
+		Given the User wants to publish a Resource
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:27]: Isn't this in backgroud? (all cases)

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 41]
```diff
@@ -0,0 +1,55 @@
...
+		And the Editor has restricted publishing rights to Curators
+		When the Curator approves the Request
+		Then the Request status is changed to "Approved"
+		And the Resource is published
+
+	@needJiraTag
+	Scenario: Curator approves an Approval Request to grant open access to content
+		Given the User has published a Resource
+		And the Editor demands a Approval to open access to Resources' content
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:36]: an Approval…the? Resource's content

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 42]
```diff
@@ -0,0 +1,55 @@
...
+	@needJiraTag
+	Scenario: Curator approves an Approval Request to publish
+		Given the User wants to publish a Resource
+		And the Editor has restricted publishing rights to Curators
+		When the Curator approves the Request
+		Then the Request status is changed to "Approved"
+		And the Resource is published
+
+	@needJiraTag
+	Scenario: Curator approves an Approval Request to grant open access to content
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:34]: Do you mean Open Access, or "free access"?

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 43]
```diff
@@ -0,0 +1,55 @@
...
+		Given the User has published a Resource
+		And the Editor demands a Approval to open access to Resources' content
+		When the Curator approves the Request
+		Then the Request status is changed to "Approved"
+		And the the Resources' content is open accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:39]: resource's…freely accessible

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 44]
```diff
@@ -0,0 +1,55 @@
...
+		And the the Resources' content is open accessable 
+		# Don't remove the potentially original embargo on the files
+
+	@needJiraTag
+	Scenario: Curator declines access to content
+		Given the User has published a Resource
+		And the Editor demands a Approval to open access to Resources' content
+		When the Curator declines the Request
+		Then the Request status is changed to "Declined"
+		And the content of the Resource is still not public accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:48]: Here we say publicly…probably this?

Comment (brinxmat) [reply 1223-curator_worklist/my_worklist_approval.feature:48]: accessible

File: 1223-curator_worklist/my_worklist_approval.feature (hunk 42 above)
Comment (brinxmat) [reply 1223-curator_worklist/my_worklist_approval.feature:34]: Or publicly as below (which is my favourite, right now)

File: 1223-curator_worklist/my_worklist_ownership.feature [hunk 45]
```diff
@@ -0,0 +1,39 @@
...
+    As a User 
+    I want to move ownership to my other account
+
+    In order to preserve access to content published at this Institution
+    As a Editor
+    I want a Curator to approve all changes of ownership of Resources
+
+	Background: 
+		Given a User has created a Request of Ownership
+		And the Curator views the landing page of the Resource assosiated with the Request
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_ownership.feature:21]: associated

File: 1223-curator_worklist/my_worklist_ownership.feature [hunk 46]
```diff
@@ -0,0 +1,39 @@
...
+    In order to manage my own work that is currently owned by my other account at some other Institution
+    As a User 
+    I want to move ownership to my other account
+
+    In order to preserve access to content published at this Institution
+    As a Editor
+    I want a Curator to approve all changes of ownership of Resources
+
+	Background: 
+		Given a User has created a Request of Ownership
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_ownership.feature:20]: an Ownership Request

File: 1223-curator_worklist/my_worklist_ownership.feature [hunk 47]
```diff
@@ -0,0 +1,39 @@
...
+		And the Curator views the landing page of the Resource assosiated with the Request
+
+	@needJiraTag
+	Scenario: Curator declines an Ownership Request
+		When the Curator declines a Request 
+		Then the Request Status is set to "Declined"
+		And the Curator may send a reason to the Request Sender
+
+	@needJiraTag
+	Scenario: Curator approves a Ownership Request
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_ownership.feature:30]: an

File: 1223-curator_worklist/my_worklist_ownership.feature [hunk 48]
```diff
@@ -0,0 +1,39 @@
...
+		When Curator approves the Request
+		Then the Request status is "Approved"
+		And the Resource's Owner is changed to the Requests' Sender
+		And the Resource's Institution is changed to the Requests' Senders' Institution
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_ownership.feature:34]: And the Resource's Institution is changed to the Request Sender's Institution

File: 1223-curator_worklist/my_worklist_search.feature [hunk 49]
```diff
@@ -0,0 +1,30 @@
...
+        And My Worklist contains Requests of each type:
+            | Approval  | 
+            | Support   | 
+            | DOI       | 
+            | Ownership |
+
+    @2795
+    @updated
+    Scenario Outline: Curator searches in Requests from My Worklist
+        When a Curator searches for a term that match at least one Request in their Worklist
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_search.feature:15]: matches

File: 1223-curator_worklist/my_worklist_search.feature [hunk 50]
```diff
@@ -0,0 +1,30 @@
...
+        Then the Curator sees "<RequestStatus>" listed as "<Status>" Checkboxes
+        And the search results is presented in Request Type tabs:
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_search.feature:17]: are presented

File: 1223-curator_worklist/readme [hunk 51]
```diff
@@ -0,0 +1,24 @@
+The Worklist is the Curator's main tool to follow up on Requests from the Users. The Curators share the Worklist and the responsibility to follow up all Requests. Each Request is part of a drafted or published Resource.
+
+Curator Scope
+The Editor can narrow a curator's responsibility to any formal part of the Institusion like an faculty, institute or a reserch group. The Curator may override this and view other Requests at the Institution and thus assist other Curators.
```
Comment (brinxmat) [inline 1223-curator_worklist/readme:4]: Institution

File: 1226-editor/readme [hunk 52]
```diff
@@ -0,0 +1,20 @@
+The Editor is the main custodian of the business aspects of the Institutional repository. The Administrator is their technical counterpart. 
+An Institution must have both roles assigned to be operational.
+
+Any editor is the Editor
+The Editor may grant the Editor role to other Users, at the same Institution. There is no built-in support for sharing responsibibility between Editors. The Editors decide how many Curators the Institution needs.
+
+Editor is the teamlead of the Curators
+A Curator's default responsibility is all Users at the Institution, but the Editor may narrow it to any formal part of the Institusion and thus reduce the group of Users this Curator is responsible for. Any unresolved conflicts or requests from own Users, or conflicts involving Users from other Institutions, might be an Editor's task. It is advised that the Editor User is also assigned the Curator role.
```
Comment (brinxmat) [inline 1226-editor/readme:8]: Institution

File: 1226-editor/readme [hunk 53]
```diff
@@ -0,0 +1,20 @@
...
+
+How many Editors and Curators?
+Smaller Institutions may deem one User to be both the Editor and the Curator. A larger institution may need a handful of Curators. A large Institution may deem it necessary to define each second-to-top level Curator an Editor and each Editor may have several Curators. Some reasearch teams may create enoungh work to need their own Curators.
+
+Use of Vocabularies
+Some Institutions wish to arrange resources into classes or categories according to shared qualities or characteristics. To accommodate this goal, the Editor may decide which vocabularies the Users may use in the Registration Wizard. Vocabularies are field-/domain-specific, so Institutions with a strong field focus may wish all Users to use the same vocabulary. Others may offer the User a selection of optional vocabularies.
+
+Publishing permissions
+Any logg-in User can publish a resource, unless the Editor has set permissions to either:
```
Comment (brinxmat) [inline 1226-editor/readme:17]: logged-in

File: 1228-landing_page/landing_page_authenticated_user.feature [hunk 54]
```diff
@@ -0,0 +1,11 @@
+Feature: Authenticated User opens a Resources Landing Page
```
Comment (brinxmat) [inline 1228-landing_page/landing_page_authenticated_user.feature:1]: Resource's

File: 1228-landing_page/landing_page_authenticated_user.feature [hunk 55]
```diff
@@ -0,0 +1,11 @@
...
+
+	Background:
+		Given a User has logged in
+		And the User opens a Resource's Landing Page
+
+	@needJiraTag
+	Scenario: Authenticated User claim ownership of the Resource
```
Comment (brinxmat) [inline 1228-landing_page/landing_page_authenticated_user.feature:8]: claims

Comment (brinxmat) [reply 1228-landing_page/landing_page_authenticated_user.feature:8]: a Resource

File: 1228-landing_page/landing_page_authenticated_user.feature [hunk 56]
```diff
@@ -0,0 +1,11 @@
...
+		When the User claims ownership of the Resource
```
Comment (brinxmat) [inline 1228-landing_page/landing_page_authenticated_user.feature:9]: a Resource

File: 1228-landing_page/landing_page_authenticated_user.feature [hunk 57]
```diff
@@ -0,0 +1,11 @@
...
+		Then a Ownership Request is created
+		And it contains a Message where the User presents his claim
```
Comment (brinxmat) [inline 1228-landing_page/landing_page_authenticated_user.feature:11]: their claim

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by brinxmat — APPROVED** [review]

**Review by terjehell — APPROVED** [review]

**Pull request merged**